UKHomeOffice/sgmr-service
# COP-4305: Fix an issue when ports are cleared when moving between steps

## Description

Fix an issue when the arrival and departure ports are cleared when moving between the voyage form steps.

## Testing

1. Go to the voyage form
2. Fill the departure details and continue to the next step
3. Fill the arrival details and continue to the next step
4. Go to the arrival and departure steps and check if the port fields are still filled

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
